core/src/Cuda/Kokkos_Cuda_Instance.cpp

```diff
@@ -822,62 +822,24 @@ Cuda::size_type Cuda::device_arch() {
 void Cuda::impl_finalize() { Impl::CudaInternal::singleton().finalize(); }
 
 Cuda::Cuda()
-    : m_space_instance(&Impl::CudaInternal::singleton()), m_counter(nullptr) {
+    : m_space_instance(&Impl::CudaInternal::singleton(), /*owning*/ false,
```

Comment:
If you put `owning=` in the comment instead of just `owning`, there's a clang-tidy check that will check that this is the right parameter.